src/com/company/Company.java

```diff
@@ -0,0 +1,125 @@
+package com.company;
+
+public class Company {
+
+    private Manager[] managers;
+    private SalesMan[] salesMan;
+    private Decorator[] decorators;
+    private int salaryManager;
+    private int salarySalesMans;
+    private int salaryDecorators;
+
+    /**
+     * @param manager   - array with data about managers;
+     * @param salesMan  - array with data about salesMans;
+     * @param decorator - array with data about decorator;
+     */
+    public Company(Manager[] manager, SalesMan[] salesMan, Decorator[] decorator) {
+        this.managers = manager;
+        this.salesMan = salesMan;
+        this.decorators = decorator;
+    }
+
+    /**
+     * @return - count manager salary;
+     */
+    public int countManagerSalary() {
```

Comment:
You can replace all 3 methods with one `int calculateSalary(Employee[] employees)`